### PR TITLE
Add Colors for 500-1000 and above 2000

### PR DESCRIPTION
**Pull request by SnejPro**
I changed orange to yellow for incidence < 50.
Then I moved all subsequent colors by one and created a new range between 500 and 1000.
At > 2000 I increased the opacity of black so you can see a difference.

Maybe a legend should be added.

### CODE REVIEW

**Comment by SnejPro** [comment]
So now i've created a legend.

File: web/assets/script.js [hunk 1]
```diff
@@ -87,16 +87,21 @@ function render(data, counties)
...
                 color = "darkred";
             else if(entry.incidence <= 1000)
                 color = "purple";
+            else if(entry.incidence <= 2000)
+                color = "black";
+                fillOpacity = 0.5
             else
                 color = "black";
-                fillOpacity = 0.6
+                fillOpacity = 0.8
```
Comment (M4GNV5) [inline web/assets/script.js]: the `fillOpacity = 0.5` statement is not part of the `else if` and thus results in the following else to be a syntax error.

File: web/index.html [hunk 2]
```diff
@@ -58,14 +67,26 @@
...
+			</div>
+			<div class="info-col">
+				<span style="color:green">⬤</span> &lt; 35<br>
+				<span style="color:yellow">⬤</span> 35 - 50<br>
+				<span style="color:orange">⬤</span> 50 - 100<br>
+				<span style="color:red">⬤</span> 100 - 200<br>
+				<span style="color:darkred">⬤</span> 200 - 500<br>
+				<span style="color:purple">⬤</span> 500 - 1000 <br>
+				<span style="color:#595959;">⬤</span> 1000 - 2000<br>
+				<span style="color:black">⬤</span> &gt; 2000<br>
```
Comment (M4GNV5) [inline web/index.html]: I like the idea of having a legend, but the map overlays are already pretty big on mobile.
Having a legend probably blocks large parts of phone screens. Maybe we should hide it on mobile using CSS? Or even make it toggable?

File: web/index.html [hunk 3]
```diff
@@ -32,6 +32,15 @@
                 left: 2px;
                 bottom: 2px;
                 z-index: 99999;
+				display: flex;
+            }
+		
+			.info-col {
```
Comment (M4GNV5) [inline web/index.html]: You are mixing spaces and tabs here. Please keep the indentation style consistent.

**Review by M4GNV5 — CHANGES_REQUESTED** [review]

**Review by M4GNV5 — APPROVED** [review]
Looks good now :)
Thanks for the input.

File: web/assets/script.js (hunk 1 above)
Comment (SnejPro) [reply web/assets/script.js]: Fixed, thank you. I wasn't aware of this because i alwayse use brackets

**Comment by SnejPro** [comment]
> Looks good now :) Thanks for the input.

The legend does not look good. I'm going to have a look in this.

**Comment by SnejPro** [comment]
@M4GNV5 Where can i find the `county-polygons-bavaria.json`? I'm trying to test on my own server.

**Comment by SnejPro** [comment]
> @M4GNV5 Where can i find the `county-polygons-bavaria.json`? I'm trying to test on my own server.

Nevermind. I found it.

**Comment by M4GNV5** [comment]
You can also test locally by having a local webserver. E.g. `python3 -m http.server` or something like `XAMPP` on Windows.

**Pull request merged**